Dbhelpers/noteHelpers.js

```diff
@@ -0,0 +1,41 @@
+const db = require('../dbConfig');
+
+getNotes = () => {
+    return db('notes')
+};
+
+
+addNote = (note) => {
+    return db('notes')
+        .insert(note)
+        .then(ids => ({ id: ids[0] })
+
+        )
+};
+
+getNote = (id) => {
```

Comment:
Very close to the var name of your other helper function. I would change it up a little to reduce the risk of calling the wrong function.